package.json

```
@@ -18,29 +18,42 @@
 		"GitHub",
 		"GraphQL"
 	],
-	"extensionDependencies": ["joyceerhl.vscode-graphql-notebook"],
+	"extensionDependencies": [
+		"joyceerhl.vscode-graphql-notebook"
+	],
 	"activationEvents": [
 		"onNotebook:gqlnb",
 		"onNotebook:github-graphql-nb"
 	],
 	"browser": "./dist/browser/extension",
 	"main": "./dist/extension",
 	"contributes": {
+		"configuration": {
+			"title": "GitHub GraphQL Notebooks",
+			"properties": {
+				"github-graphql-nb.scopes": {
+					"type": "array",
+					"default": [ "repo", "workflow" ],
+					"markdownDescription": "The authentication scopes to request when signing into GitHub. See [Available Scopes](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes) in Github Documentation for more information.",
```

Nice description :clap: I appreciate the link to the documentation! 

```suggestion
					"markdownDescription": "The authentication scopes to request when signing into GitHub. See [Available Scopes](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes) in GitHub Documentation for more information.",
```